scripts/makebumpver

```diff
@@ -247,21 +232,12 @@ class MakeBumpVer:
     def _queryBug(self, bugid):
         if not self.bz:
             sys.stdout.write("Connecting to %s...\n" % self.bzserver)
-
-            if not self.username:
-                sys.stdout.write('Username: ')
-                self.username = sys.stdin.readline()
-                self.username = self.username.strip()
-
-            if not self.password:
-                self.password = getpass.getpass()
-
-            bzclass = bugzilla.Bugzilla
-            self.bz = bzclass(url=self.bzurl)
+            self.bz = bugzilla.Bugzilla(url=self.bzurl)
 
         if not self.bz.logged_in:
-            rc = self.bz.login(self.username, self.password)
-            log.debug("login rc = %s", rc)
+            msg = "Provide an API key from https://{}/userprefs.cgi?tab=apikey\n"
+            sys.stdout.write(msg.format(self.bzserver))
```

Comment:
I think it might make more sense to have the format on same line as the string literal, so that it can be checked.

Reply:
Fixed.